RELEASING.md

```diff
@@ -26,20 +26,21 @@ TIP: To get the short ID for your key `gpg2 --show-keys --keyid-format short ${Y
 
 ## Running a release
 
-The project is release is two parts, the API and the Framework. There are separate release scripts beneath `./scripts` for each of these.
+Use the release script to actually perform the release and prepare main for the next development version.
 
 ```shell
-scripts/release-api.sh ${RELEASE_VERSION}
+./scripts/release.sh -k <YOUR_KEY_SHORT_ID> -v <RELEASE_VERSION> [-b <BRANCH_FROM>] [-r <REPOSITORY>]
 ```
 
-```shell
-scripts/release-framework.sh ${RELEASE_VERSION}
-```
+where
+* `<YOUR_KEY_SHORT_ID>` is the short id of your PGP key 
```

Comment:
I think you mean "GPG" key

Reply:
PGP is the *requirement*,  but users are actually free to use pgp, gpg or any other pgp compatible.  We talk about PGP higher up in the doc, so this new text is consistent.   

https://www.gnupg.org/
> GnuPG is a complete and free implementation of the OpenPGP standard as defined by [RFC488](https://www.ietf.org/rfc/rfc4880.txt)






OpenPGP